packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/review/eventgrid.api.md

```diff
@@ -34,7 +35,7 @@ export interface BrokerProperties {
 // @public
 export interface CloudEvent {
     data?: any;
-    dataBase64?: Uint8Array;
+    data_base64?: Uint8Array;
```

Comment:
I am a fan of that we should do property normalization especially for azure scope. This could help us generate more idiomatic SDKs. 

TypeSpec authors could have their own style of property naming convention and that style may or may not be suitable for JS/TS language style. So property normalization could help 
- ensure all properties only have one style
- property naming style is JS/TS style
- good property name could be better discoverable by user intellisense
- A typespec property may not be a valid ts property